src/services/twilio.ts

```diff
@@ -2,29 +2,33 @@ import twilio from "twilio"
 
 import { getTwilioConfig } from "@config"
 import {
+  ExpiredOrNonExistentPhoneNumber,
   InvalidPhoneNumberPhoneProviderError,
   PhoneProviderConnectionError,
   RestrictedRegionPhoneProviderError,
   UnknownPhoneProviderServiceError,
   UnsubscribedRecipientPhoneProviderError,
 } from "@domain/phone-provider"
 import { baseLogger } from "@services/logger"
-import { wrapAsyncFunctionsToRunInSpan } from "@services/tracing"
+
+import { VerificationCheckInstance } from "twilio/lib/rest/verify/v2/service/verificationCheck"
+
+import { CodeInvalidError } from "@domain/authentication/errors"
+
+import { wrapAsyncFunctionsToRunInSpan } from "./tracing"
 
 export const TwilioClient = (): IPhoneProviderService => {
   const client = twilio(getTwilioConfig().accountSid, getTwilioConfig().authToken)
+  const verify = client.verify.v2.services(getTwilioConfig().verifyService)
 
-  const sendText = async ({ body, to, logger }: SendTextArguments) => {
-    const twilioPhoneNumber = getTwilioConfig().twilioPhoneNumber
+  const initiateVerify = async (to: PhoneNumber) => {
     try {
-      await client.messages.create({
-        from: twilioPhoneNumber,
-        to,
-        body,
-      })
+      await verify.verifications.create({ to, channel: "sms" })
```

Comment:
shouldn't channel be related/linkend to verifyService?

Reply:
not sure to understand your question